lib/Driver/Compilation.cpp

```diff
@@ -727,6 +731,15 @@ namespace driver {
                  : TaskFinishedResponse::StopExecution;
     }
 
+#if defined(_WIN32)
+    struct ChangeStderrToBinaryMode {
```

Comment:
WDYT of `FileBinaryModeRAII` as a name and taking a `FILE *` as a ctor parameter instead of hardcoding this?  That would make this more re-usable and name it more in-line with the LLVM scheme as well.

Reply:
SGTM and done.